frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.styled.tsx

```diff
@@ -60,7 +60,7 @@ DashboardHeaderButton.defaultProps = {
 };
 
 export const SectionMenuItem = styled(Menu.Item)`
-  background-color: ${darken(color("bg-medium"), 0.1)};
+  background-color: ${() => darken(color("bg-medium"), 0.1)};
```

Comment:
nit: I see some places we use `darken("bg-medium", 0.1)` without the additional `color` call, so we can use that instead.

Reply:
good point. That works, although it won't get the latest value from the theme if we use the color that allows overriding. In that case we need to use. `theme.fn.themeColor(...)` instead. So it will look something like

```ts
darken(theme.fn.themeColor(overriddenColor))
```

Reply:
@WiNloSt Given that we have the `setGlobalEmbeddingColors` which re-applies the colors right before rendering, I think it might've already applied the last colors. But yes, using `theme.fn.themeColor` is better as it does not rely on the hack.

```ts
  const themeOverride = useMemo(() => {
    // !! Mutate the global colors object to apply the new colors.
    // This must be done before ThemeProvider calls getThemeOverrides.
    setGlobalEmbeddingColors(theme?.colors, appColors);

    return theme && getEmbeddingThemeOverride(theme);
  }, [appColors, theme]);
```

Reply:
that's new to me. Good to know we have this fallback at least. 👍